### PR TITLE
Scrubs docs about zipkin-web as it is now embedded

### PR DESCRIPTION
**Pull request by adriancole**

### CODE REVIEW

**Comment by adriancole** [comment]
@dsyer PTAL I know we don't need zipkin-web anymore, but not sure if we should have similar CF instructions for zipkin-server or not

**Comment by marcingrzejszczak** [comment]
Hmm if we have spring-cloud-stream then we need a UI of some sort. That's why we also deploy Zipkin Web

**Comment by adriancole** [comment]
zipkin's UI is now hosted on the zipkin server itself! so basically, if you
hit http://localhost:9411 you'll see the UI now.

Here's the spring config:

https://github.com/openzipkin/zipkin-java/blob/master/zipkin-server/src/main/java/zipkin/server/ZipkinUiConfiguration.java#L40

File: spring-cloud-sleuth-zipkin-stream/README.md [hunk 1]
```diff
@@ -24,24 +24,11 @@ public class ZipkinStreamServerApplication {
 }
 ```
 
+Zipkin has a web UI, which is enabled by default when you depend on `io.zipkin:zipkin-ui`.
```
Comment (adriancole) [inline spring-cloud-sleuth-zipkin-stream/README.md]: @marcingrzejszczak wdyt about this?

Comment (marcingrzejszczak) [inline spring-cloud-sleuth-zipkin-stream/README.md]: Sweet!

**Pull request merged**